internal/statemachine/classic_test.go

```diff
@@ -1476,15 +1476,21 @@ func TestSuccessfulClassicRun(t *testing.T) {
 		saveCWD := helper.SaveCWD()
 		defer saveCWD()
 
+		// We need the output directory set for this
+		outputDir, err := ioutil.TempDir("/tmp", "ubuntu-image-")
+		asserter.AssertErrNil(err, true)
+		defer os.RemoveAll(outputDir)
+
 		var stateMachine ClassicStateMachine
 		stateMachine.commonFlags, stateMachine.stateMachineFlags = helper.InitCommonOpts()
 		stateMachine.parent = &stateMachine
 		stateMachine.commonFlags.Debug = true
 		stateMachine.commonFlags.Size = "4G"
+		stateMachine.commonFlags.OutputDir = outputDir
 		stateMachine.Args.ImageDefinition = filepath.Join("testdata", "image_definitions",
 			"test_amd64.yaml")
 
-		err := stateMachine.Setup()
+		err = stateMachine.Setup()
```

Comment:
This seemed unrelated at first, but I see now that `err` is being declared above (`outputDir, err := ioutil.TempDir("/tmp", "ubuntu-image-")`), so having just the (re-)assignment down here is good.